src/include/aerospike/as_exp.h

```diff
@@ -2972,7 +2974,7 @@ as_exp_destroy_base64(char* base64)
  * @param Bin to which the application applies.
```

Comment:
This is missing the parameter names; the other expression macros have them

For `__rtype`, it might help to include the enum name so downstream developers know what the valid values are:

` * @param __vtype		Value type (as_exp_type).`